Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -875,7 +875,7 @@ void RenderLayer::collectLayers(std::unique_ptr<Vector<RenderLayer*>>& positiveZ
 
     bool isStacking = isStackingContext();
     // Overflow layers are just painted by their enclosing layers, so they don't get put in zorder lists.
```

Comment:
Should this comment be removed now too?

Reply:
Digging back through history, it seems this comment is meaning the `!isNormalFlowOnly()` check.

Maybe I should change 'Overflow layers' to 'normal flow layers'?

Reply:
Terminology changed here: https://commits.webkit.org/33236@main